backend/Lingva.BusinessLayer/Services/MemberService.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Lingva.DataAccessLayer.Entities;
+using Microsoft.AspNetCore.Mvc;
+using Lingva.DataAccessLayer.Repositories;
+using Lingva.DataAccessLayer.Context;
+
+
+namespace UsersManagementBackEnd.Services
+{
+    public class EventMemberService : IMembersService
+    {
+        public ApplicationContext _context;
+        RepositoryEventMember repositoryEventMember;
+
+        public EventMemberService()
+        {
+            _context = new ApplicationContext();
```

Comment:
We don't need to create ApplicationContext explicity.
Use a constructor injection and configure one an ASP.NET Core side in ServiceExtensions.